crates/gen/src/types/method.rs

```diff
@@ -67,7 +67,7 @@ impl Method {
         let return_type = if blob.read_expected(0x01) {
             None
         } else {
-            let name = "__result".to_owned();
+            let name = "result__".to_owned();
```

Comment:
Switched from leading to trailing underscores for internal generated names to allow unused variables to quickly be detected.